### DISCUSSION TITLE
Split overview from workspace view

### DISCUSSION THREAD

**Discussion by itouchdennis** [end-4/dots-hyprland#575] [Q&A]
Hi there!

At first:
I just seen this project since 555 nvidia drivers out and its huge! Thanks for the work!

Now my questions:
Is it possible to split the ags worksation view from the search bar?
So e.g. I hit "super" and only the search bar appears, and if I hit ALT_L+TAB the workspace view pops up.

(Also, the icons view in the workspace overview is bugged as I am using a 21:9 display its looking weird at the first time as the icons are HUGE AF, Idc. what I changed now, but currently the icons in between the workspace views are completely empty / not even shown.)

TY!

**Comment by itouchdennis** [comment]
Found out the icons issue seems to be related to my 2. display.
If I turn it off the icons are missing in the overview, if I turn it on, icons are there.

**Comment by itouchdennis** [comment]
![2024-06-03-102541_hyprshot](https://github.com/end-4/dots-hyprland/assets/12660642/31fcf86e-5317-483b-92b0-804e9bc19487)

Thats how the icons look like when both displays are on, its a bit off - I guess its about the 3440x1440 res.

**Comment by end-4** [comment]
> Is it possible to split the ags worksation view from the search bar?

Nope, it's like that by design and there's no simple configuration to achieve that

---

As for the icons, yeah the overview isn't very multimonitor-friendly...
